lib/Overlay/Overlay.less

```diff
@@ -0,0 +1,88 @@
+
+.moon-overlay {
+	position: relative;
+}
+
+.moon-overlay-component {
+	display: inline-block;
+	pointer-events: auto;
+}
+
+.moon-overlay-container {
+	position: absolute;
+	top: 0;
+	right: 0;
+	bottom: 0;
+	left: 0;
+	padding: 12px;
+	background-color: rgba(0, 0, 0, 0.5);
+	pointer-events: none;
+	display: none;
+
+	&.position-top { bottom: auto; }
+	&.position-bottom { top: auto; }
+	&.position-left { right: auto; }
+	&.position-right { left: auto; }
+
+	&.align-right { text-align: right; }
+	&.align-left { text-align: left; }
+
+	&.transparent {
+		background-color: transparent;
+	}
+
+	&.centered .moon-overlay-component {
+		position: absolute;
+		top: 50%;
+		left: 50%;
+		-webkit-transform: translateX(-50%) translateY(-50%);
+		transform: translateX(-50%) translateY(-50%);
+	}
+
+	&.moon-overlay-selection .moon-icon {
+		display: block;
+		width: @moon-icon-button-small-size;
+		height: @moon-icon-button-small-size;
+		line-height: @moon-icon-button-small-size;
+		border: @moon-button-border-width solid black;
+		background-color: white;
+		color: transparent;
+		margin: 0;
+		border-radius: 50%;
+		position: relative;
+		opacity: 0.5;
+
+		.selected & {
```

Comment:
WOW! That's convenient! I hadn't thought to use it like that before. I'll for sure be taking advantage of that in the future, especially for all of our `.enyo-locale-right-to-left` and locale classes.